extensions/amp-brightcove/1.0/amp-brightcove.css

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Pre-upgrade:
+ * - display:block element
+ * - size-defined element
+ */
+amp-brightcove {
+  display: block;
+  overflow: hidden;
```

Comment:
FMI: why are `overflow: hidden` and `position: relative` needed pre-upgrade? How could a valid AMP Page have anything overflow?

Reply:
- `relative` is required for the `VideoWrapper` being contained in the box.
- `block` because it's otherwise `inline-block`. I'm not sure if we would cause a layout shift if removed.

Reply:
and overflow?

Reply:
`overflow: hidden` prevents the internal tree from spilling outside the container's box.